Index: src/main/java/ru/learningJava/homework3/App.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework3/App.java b/src/main/java/ru/learningJava/homework3/App.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework3/App.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,23 +0,0 @@
-package ru.learningJava.homework3;
-
-import java.util.Scanner;
-
-public class App {
-    public static void main(String[] args) {
-
-        TV LG = new TV("LG",false,40.5);
-        Scanner scaner = new Scanner(System.in);
-        LG.setScreenFrequency(ScreenFrequncy.getRandomFrequncy());
-        LG.setScreenResolution("HD");
-
-        TV panasonic = new TV("Panasonic", true,108);
-        panasonic.setScreenFrequency(ScreenFrequncy.getRandomFrequncy());
-        panasonic.setScreenResolution("2K");
-
-
-        System.out.println(LG);
-        System.out.println();
-        System.out.println(panasonic);
-
-    }
-}
Index: src/main/java/ru/learningJava/homework2/ArraySymbol.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework2/ArraySymbol.java b/src/main/java/ru/learningJava/homework2/ArraySymbol.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework2/ArraySymbol.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,13 +0,0 @@
-package ru.learningJava.homework2;
-
-public class ArraySymbol {
-    public static void main(String[] args) {
-
-        for (int i=0; i<10; i++) {
-            System.out.println();
-            for (int j = 0; j < 10; j++) {
-                System.out.print("-");
-            }
-        }
-    }
-}
Index: src/main/java/ru/learningJava/homework2/fahrenheitToCelsius.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework2/fahrenheitToCelsius.java b/src/main/java/ru/learningJava/homework2/fahrenheitToCelsius.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework2/fahrenheitToCelsius.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,23 +0,0 @@
-package ru.learningJava.homework2;
-
-import java.util.Scanner;
-
-public class fahrenheitToCelsius {
-
-    public static void main(String[] args) {
-        Scanner readerFahrenheitValue = new Scanner(System.in);
-
-        System.out.println("Введите значение температуры по шкале Фаренгейта");
-        double fahrenheitValue = readerFahrenheitValue.nextDouble();
-        System.out.println("Введенное значение эквивалентно " + fToc(fahrenheitValue) + "градусам цельсия");
-
-    }
-
-
-    public static double fToc(double fahrenheitValue) {
-        double celsiusValue;
-        celsiusValue = (fahrenheitValue - 32) * 5/9;
-        return celsiusValue;
-    }
-
-}
Index: src/main/java/ru/learningJava/homework4/FindArrow.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework4/FindArrow.java b/src/main/java/ru/learningJava/homework4/FindArrow.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework4/FindArrow.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,20 +0,0 @@
-package ru.learningJava.homework4;
-
-public class FindArrow {
-    public static void main(String[] args) {
-        String text = "<--<<>>-->>>-<--<<->>>--><-<--<<-<<";
-        String text2 = text;
-        String leftArrow = "<--<<";
-        String rightArrow = ">>-->";
-        int i = 0;
-        while (text.indexOf(leftArrow) != -1) {
-            text = text.replaceFirst(leftArrow, " ");
-            i++;
-        }
-        while (text2.indexOf(rightArrow) != -1) {
-            text2 = text2.replaceFirst(rightArrow, " ");
-            i++;
-        }
-        System.out.println(i);
-    }
-}
Index: src/main/java/ru/learningJava/homework1/FourNumbers.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework1/FourNumbers.java b/src/main/java/ru/learningJava/homework1/FourNumbers.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework1/FourNumbers.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,22 +0,0 @@
-package ru.learningJava.homework1;
-
-import java.util.Random;
-
-public class FourNumbers {
-    public static void main(String[] args) {
-        System.out.println("2   3   4   5");
-        System.out.println(getFormatString(11,22,34,46));
-
-        for (int i = 0; i < 4; i++){
-            Random randomNumber = new Random();
-            int number = randomNumber.nextInt(1,10);
-            System.out.printf("%d ",number);
-        }
-    }
-
-    private static String getFormatString(int firstNumber, int secondNumber, int thirdNumber, int fourthNumber){
-        return String.format("%s %s %s %s", firstNumber, secondNumber,thirdNumber,fourthNumber);
-    }
-
-
-}
Index: src/main/java/ru/learningJava/practice/Fruit.java
===================================================================
diff --git a/src/main/java/ru/learningJava/practice/Fruit.java b/src/main/java/ru/learningJava/practice/Fruit.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/practice/Fruit.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,13 +0,0 @@
-package ru.learningJava.practice;
-
-public class Fruit {
-    String name;
-    String color;
-    public Fruit(String name) {
-        this.name=name;
-    }
-
-    public Fruit(){
-        this.name = "Zal";
-    }
-}
Index: src/main/java/ru/learningJava/homework4/LeftCharKeyboard.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework4/LeftCharKeyboard.java b/src/main/java/ru/learningJava/homework4/LeftCharKeyboard.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework4/LeftCharKeyboard.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,22 +0,0 @@
-package ru.learningJava.homework4;
-
-
-import java.util.Scanner;
-
-public class LeftCharKeyboard {
-    public static void main(String[] args) {
-
-        String keyboardChar = "qwertyuiopasdfghjklzxcvbnm";
-
-        Scanner scaner = new Scanner(System.in);
-        String innerChar = scaner.next();
-
-        int leftCharIndex = keyboardChar.indexOf(innerChar)-1;
-        if (leftCharIndex != -1){
-            System.out.println(keyboardChar.charAt(leftCharIndex));
-        }
-        else System.out.println("m");
-    }
-
-
-}
Index: src/main/java/ru/learningJava/practice/Main.java
===================================================================
diff --git a/src/main/java/ru/learningJava/practice/Main.java b/src/main/java/ru/learningJava/practice/Main.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/practice/Main.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,12 +0,0 @@
-package ru.learningJava.practice;
-
-public class Main {
-    public static void main(String[] args) {
-        Fruit apple = new Fruit("Apple");
-        System.out.println(apple.name);
-        Fruit mango = new Fruit();
-        System.out.println(mango.name);
-        mango.color = "green";
-        System.out.println(mango.color);
-    }
-}
Index: src/main/java/ru/learningJava/homework3/ScreenFrequncy.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework3/ScreenFrequncy.java b/src/main/java/ru/learningJava/homework3/ScreenFrequncy.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework3/ScreenFrequncy.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,29 +0,0 @@
-package ru.learningJava.homework3;
-
-import java.util.Random;
-
-public enum ScreenFrequncy {
-    frequncy_60(60),
-    frequncy_75(75);
-
-    public int getFrequncyValue() {
-        return frequncyValue;
-    }
-
-    private int frequncyValue;
-    @Override
-    public String toString() {
-        return Integer.toString(frequncyValue);
-    }
-
-    ScreenFrequncy(int frequncyValue) {
-        this.frequncyValue = frequncyValue;
-    }
-
-    public static ScreenFrequncy getRandomFrequncy(){
-        Random random = new Random();
-        ScreenFrequncy[] values = ScreenFrequncy.values();
-        int randomIndex = random.nextInt(values.length);
-        return values[randomIndex];
-    }
-}
Index: src/main/java/ru/learningJava/homework4/SortChar.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework4/SortChar.java b/src/main/java/ru/learningJava/homework4/SortChar.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework4/SortChar.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,16 +0,0 @@
-package ru.learningJava.homework4;
-
-import java.util.Arrays;
-
-public class SortChar {
-    public static void main(String[] args) {
-        String text = "sdrea troid uuitr";
-        String[] words = text.split(" ");
-        for(String word : words) {
-            char[] unsortWord = word.toCharArray();
-            Arrays.sort(unsortWord);
-            System.out.println(unsortWord);
-        }
-
-    }
-}
Index: src/main/java/ru/learningJava/homework1/StoneScissorsPaper.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework1/StoneScissorsPaper.java b/src/main/java/ru/learningJava/homework1/StoneScissorsPaper.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework1/StoneScissorsPaper.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,28 +0,0 @@
-package ru.learningJava.homework1;
-
-import java.util.ArrayList;
-import java.util.List;
-
-public class StoneScissorsPaper {
-    public static void main(String[] args) {
-        int vasya = (int) (Math.random()*3);
-        int petya = (int) (Math.random()*3);
-
-        List<String> fingerSigns = new ArrayList<>();
-        fingerSigns.add("Камень");
-        fingerSigns.add("Ножницы");
-        fingerSigns.add("Бумага");
-
-        System.out.println("Vasya-"+ fingerSigns.get(vasya) + ", Petya-" + fingerSigns.get(petya));
-
-        if (vasya == petya) {System.out.println("Ничья"); System.exit(0);}
-
-        if (vasya == 0) if (petya == 1) System.out.println("Победил Vasya!");
-            else System.out.println("Победил Petya");
-        if (vasya == 1) if (petya == 2) System.out.println("Победил Vasya!");
-            else System.out.println("Победил Petya");
-        if (vasya == 2) if (petya == 0) System.out.println("Победил Vasya!");
-            else System.out.println("Победил Petya");
-
-    }
-}
Index: src/main/java/ru/learningJava/homework2/StringLoop.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework2/StringLoop.java b/src/main/java/ru/learningJava/homework2/StringLoop.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework2/StringLoop.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,11 +0,0 @@
-package ru.learningJava.homework2;
-
-public class StringLoop {
-    public static void main(String[] args) {
-        String loopedWord = "";
-        for (int i = 0; i < 8; i++){
-            loopedWord = loopedWord + "Java";
-        }
-        System.out.println(loopedWord);
-    }
-}
Index: src/main/java/ru/learningJava/homework3/TV.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework3/TV.java b/src/main/java/ru/learningJava/homework3/TV.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework3/TV.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,49 +0,0 @@
-package ru.learningJava.homework3;
-
-import java.util.Random;
-
-public class TV {
-    private  String brandName;
-    private int channelCount;
-    private boolean isColored;
-    private double diagonalSize;
-    private String screenResolution;
-
-    public ScreenFrequncy getScreenFrequency() {
-        return screenFrequency;
-    }
-
-    public void setScreenFrequency(ScreenFrequncy screenFrequency) {
-        this.screenFrequency = screenFrequency;
-    }
-
-    private ScreenFrequncy screenFrequency;
-    private String weight;
-
-    public void setScreenResolution(String screenResolution) {
-        this.screenResolution = screenResolution;
-    }
-
-
-    public TV (String brandName, boolean isColored, double diagonalSize){
-        this.brandName = brandName;
-        this.isColored = isColored;
-        this.diagonalSize = diagonalSize;
-        Random randomNumber = new Random();
-        this.weight = String.format("%.2f",randomNumber.nextDouble(5,15));
-        this.channelCount = randomNumber.nextInt(10,100);
-    }
-
-    @Override
-    public String toString() {
-        return "TV{\n" +
-                "brandName='" + brandName + "\n" +
-                "channelCount=" + channelCount + "\n" +
-                "isColored=" + isColored + "\n" +
-                "diagonalSize=" + diagonalSize + "\n" +
-                "screenResolution='" + screenResolution + "\n" +
-                "screenFrequency='" + screenFrequency + "\n" +
-                "weight=" + weight +
-                '}';
-    }
-}
Index: src/main/java/ru/learningJava/homework2/TwoNumbers.java
===================================================================
diff --git a/src/main/java/ru/learningJava/homework2/TwoNumbers.java b/src/main/java/ru/learningJava/homework2/TwoNumbers.java
deleted file mode 100644
--- a/src/main/java/ru/learningJava/homework2/TwoNumbers.java	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
+++ /dev/null	(revision d291893a75a217897ad83903e45c0dc12cbb87e2)
@@ -1,29 +0,0 @@
-package ru.learningJava.homework2;
-
-import java.util.Scanner;
-
-import static java.lang.Math.abs;
-
-public class TwoNumbers {
-    public static void main(String[] args) {
-        Scanner getNumber = new Scanner(System.in);
-        System.out.println("Введите первое число");
-        int firstNumber = getNumber.nextInt();
-        System.out.println("Введите второе число");
-        int secondNumber = getNumber.nextInt();
-        System.out.println("Певое число = " + firstNumber + ", Второе число = " + secondNumber);
-        System.out.println("Сумма чисел = " + (firstNumber + secondNumber));
-        System.out.println("Разница чисел = " + (firstNumber - secondNumber));
-        System.out.println("Произведение чисел = " + (firstNumber * secondNumber));
-        System.out.println("Среднее значение = " + (double) (firstNumber + secondNumber) / 2);
-        System.out.println("Расстояние = " + (abs(firstNumber - secondNumber)));
-        if (firstNumber > secondNumber) {
-            System.out.println("Максимальное значение = " + firstNumber);
-            System.out.println("Минимальное значение = " + secondNumber);
-        }
-        else {
-            System.out.println("Максимальное значение = " + secondNumber);
-            System.out.println("Минимальное значение = " + firstNumber);
-        }
-    }
-}
